zboss/doc/zboss_configuration.rst

```diff
@@ -5,9 +5,26 @@ ZBOSS configuration
 
 The ZBOSS stack is distributed as a set of precompiled linkable libraries.
 There are two variants of the libraries that support different Zigbee device roles, each variant having its own configuration.
```

Comment:
So now, the variants are actually four, no? "Variant" is now referring to both the device type library versions and to certification/NCP ZBOSS versions. I think we need a different way to distinguish them.

Reply:
I'd say that we have 2 versions with 2 variants each.

Reply:
Just to make it clear: the new version has everything that's needed to do both single SoC and NCP FW development, so you can enable the `CONFIG_ZIGBEE_LIBRARY_EXPERIMENTAL` in existing Zigbee applications and they will not get the NCP code linked. For the NCP FW development, the `CONFIG_ZIGBEE_LIBRARY_NCP_DEV` is required, but it is available only in the experimental version of ZBOSS :slightly_smiling_face: